.travis.yml

```diff
@@ -13,4 +13,5 @@ script:
   - bundle exec rake db:create
   - bundle exec rake db:migrate
   - bundle exec rspec spec/models
+  - bundle exec rspec spec/controllers
```

Comment:
@kelseyhuse30 Thanks for following this pattern! I decided that it was time to run more of the suite, so I ended up doing an exclusion pattern instead of individually including the directories I wanted to run: 
```
bundle exec rspec --exclude-pattern "**/views/*/*_spec.rb, **/requests/*_spec.rb"
```